collect_app/src/main/java/org/odk/collect/android/storage/StorageEraser.java

```diff
@@ -0,0 +1,24 @@
+package org.odk.collect.android.storage;
+
+import java.io.File;
+
+class StorageEraser {
```

Comment:
I like that this is pulled out as its own object! I think you could pass the `StoragePathProvider` in as a constructor arg rather than have it passed in to both methods as it feels like a dependency.

Reply:
Done.